scripts/deploy/protocol/prepareScoutGameLaunchSafeTransaction.ts

```diff
@@ -411,37 +393,76 @@ task('prepareScoutGameLaunchSafeTransaction', 'Prepares a Safe transaction to la
       value: '0'
     };
 
-    await apiKit.estimateSafeTransaction(safeAddress, lockupCreateStreamTxData);
+    // await apiKit.estimateSafeTransaction(safeAddress, lockupCreateStreamTxData);
```

Comment:
can we leave these in, i don't know what they did be fore - maybe just make sure the gas wasnt too high?

Reply:
yeah sure that could be the case